maps/Nightmare/maps/BigRed/nightmare.json

```diff
@@ -1,16 +1,23 @@
 [
 	{ "type": "map_sprinkle", "path": "sprinkles/" },
-	{ "type": "pick",     "chance": 0.40, "choices": [
+	{ "type": "pick",     "chance": 0, "choices": [
 		{ "type": "map_insert", "landmark": "crashlanding-offices", "path": "standalone/crashlanding-offices.dmm" },
 		{ "type": "map_insert", "landmark": "crashlanding-eva",   "path": "standalone/crashlanding-eva.dmm" }
 	]},
-	{ "type": "map_insert", "chance": 0.50, "landmark": "lambda-graveyard", "path": "standalone/lambda-graveyard.dmm" },
-	{ "type": "pick",     "chance": 0.50, "choices": [
+	{ "type": "map_insert", "chance": 0, "landmark": "lambda-graveyard", "path": "standalone/lambda-graveyard.dmm" },
+	{ "type": "pick",     "chance": 0, "choices": [
 		{ "type": "map_insert", "landmark": "lambda-cave-mushroom", "path": "standalone/lambda-cave_mushroom.dmm" },
 		{ "type": "map_insert", "landmark": "lambda-cave-extratunnel",   "path": "standalone/lambda-cave_extratunnel.dmm" }
 	]},
-	{ "type": "pick",     "chance": 0.50, "choices": [
```

Comment:
seems you disabled the chance for other inserts to spawn here

Reply:
This is for my testing purposes. I'm also not sure what odds this should spawn in for.